libs/cart/testing/src/factories/cart-item.factory.ts

```diff
@@ -1,11 +1,11 @@
 import { Injectable } from '@angular/core';
 import * as faker from 'faker/locale/en_US';
 
-import { CartItem } from '@daffodil/cart';
+import { DaffCartItem } from '@daffodil/cart';
 
 import { DaffModelFactory } from "@daffodil/core/testing";
 
-export class MockCartItem implements CartItem {
+export class MockCartItem implements DaffCartItem {
```

Comment:
I missed this in the product PR, but I think this should be `MockDaffCartItem`

Reply:
So my thought here is that since `MockCartItem` isn't ever exported from the package, it doesn't need to have the prefix. Internal features don't warrant the prefix.